assets/js/admin.js

```diff
@@ -61,7 +61,9 @@ jQuery(document).ready(function($) {
     Admin settings tab switching
      */
     var currentTab;
-    if ( localStorageAvailable() && window.localStorage.getItem( 'Auth0WPSettingsTab' ) ) {
+    if ( window.location.hash ) {
```

Comment:
Use the hash if we have one so we're able to link to specific tabs in the settings panel